app/templates/verkiezingen/installatievergadering.hbs

```diff
@@ -106,6 +113,37 @@
       </div>
     {{/unless}}
   </div>
+
+  <AuModal
+    @title="Selecteer type verkiezingsuitslag"
+    @modalOpen={{this.verkiezingsUitslagModal}}
+    @closable={{true}}
+    @closeModal={{fn (mut this.verkiezingsUitslagModal) false}}
+  >
+    <div class="au-o-box">
+      Deze installatievergadering bevat de uitslag van meerdere verkiezingen.
+      Selecteer welke uitslag je wilt bekijken.
+      <li class="au-c-list-horizontal__item au-u-margin-top-small">
+        <AuDropdown
+          @title="Selecteer verkiezingsuitslag"
+          @alignment="left"
+          @skin="secondary"
+          role="menu"
+        >
+          {{#each @model.verkiezingen as |verkiezing|}}
+            <AuLink
```

Comment:
not super annoying but this modal stays open if you click back in your browser navigation. maybe you can use setupcontroller to set this.verkiezingsUitslagModal to false? If not, don't care